fineract-provider/toif/housekeeping/housekeeping.txt

```diff
@@ -0,0 +1,36 @@
+###########################
+#		  Facts
+###########################
+
+TOIFSegmentIsRelatedToProject=project1
+TOIFSegmentIsProducedByOrganization=org1
+TOIFSegmentIsOwnedByOrganization=org1
+TOIFSegmentIsGeneratedByPerson=person1
+TOIFSegmentIsSupervisedByPerson=person1
+
+PersonIsInvolvedInProjectAsRole=person1;project1;role1
+OrganizationIsInvolvedInProjectAsRole=org1;project1;role2
+OrganizationIsPartOfOrganizationAsRole=org1;org2;role2
+PersonIsEmployedByOrganizationAsRole=person1;org1;role1
+
+###########################
+#		Entities
+###########################
+
+SegmentDescription=Segment.
+
+#projectId=name;description
+project1=apache-fineract;Apache Fineract:A Platform for Microfinance.
+
+#personId=name;email;phone
+person1=mifos-developer;dev@fineract.apache.org;555-1234
```

Comment:
Suggest not to have mifos names as this is Apache product now.

Reply:
@nazeer1100126 will do it.

Reply:
@ThisuraThejith Can you update your PR with the suggested change so it can be merged?